dandelion.json

```diff
@@ -96,7 +96,7 @@
     },
     "company-oms-mail-connector": {
       "path": "bundles/company-oms-mail-connector/",
-      "version": "1.1.1"
+      "version": "2.0.0"
```

Comment:
is it a breaking change or just a new feature?

Reply:
it is a breaking change because you need to use the custom OrderConfirmationMailTypePlugin instead spryker default.